.hyperagent/census-2026-06-27/census_synthesis.py

```
@@ -0,0 +1,205 @@
+
+# ====================== SEPARATION (coverage-conditioned) ==================
+pairs = [(a,b) for a in range(N) for b in range(a+1,N)]
+jac = [cm.jaccard(COV[a],COV[b]) for a,b in pairs]
+mean_jac = statistics.mean(jac)
+# entity-set separation (MASI distance) on the same pairs
+ent = [cm.masi_distance(reader_sets[a],reader_sets[b]) for a,b in pairs]
+mean_ent = statistics.mean(ent)
+# correlation: does higher coverage overlap predict lower entity separation?
+def pearson(x,y):
+    mx,my=statistics.mean(x),statistics.mean(y)
+    num=sum((a-mx)*(b-my) for a,b in zip(x,y))
```

Ruff (0.15.18)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)